models/category/src/category-search/builder.spec.ts

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import type { TCategorySearch, TCategorySearchGraphql } from '../types';
+import * as CategorySearch from './index';
+
+describe('builder', () => {
+  it(
+    ...createBuilderSpec<TCategorySearch, TCategorySearch>(
+      'default',
+      CategorySearch.random(),
+      expect.objectContaining({
+        id: expect.any(String),
+        version: expect.any(Number),
+        key: expect.any(String),
+        externalId: expect.any(String),
+        orderHint: expect.any(String),
+        parent: expect.objectContaining({
+          typeId: 'category',
+        }),
+        name: expect.objectContaining({
+          en: expect.any(String),
+          de: expect.any(String),
+          fr: expect.any(String),
+        }),
+        description: expect.objectContaining({
+          en: expect.any(String),
+          de: expect.any(String),
+          fr: expect.any(String),
+        }),
+        slug: expect.objectContaining({
+          en: expect.any(String),
+          de: expect.any(String),
+          fr: expect.any(String),
+        }),
+        createdAt: expect.any(String),
+        lastModifiedAt: expect.any(String),
+        parentRef: expect.objectContaining({
+          typeId: 'category',
+        }),
+        childCount: expect.any(Number),
+        stagedProductCount: expect.any(Number),
+        ancestorsRef: [],
+        ancestors: [],
+        productTypeNames: [],
+        children: [],
+        assets: [],
+        custom: [],
+      })
```

Comment:
nit: I don't see this being done a lot, so I am not sure if there are reasons against it, but personally I think its easier if we set this big block into a variable and reuse it in every test, it helps understand what are the differences between rest and graphQL

Reply:
Done https://github.com/commercetools/test-data/pull/671/commits/18bd5cf9b5f4711dd329e87a5bdc7128affff1ec